packages/openapi-generator/src/parser/refs.spec.ts

```diff
@@ -78,34 +101,30 @@ describe('OpenApiDocumentRefs', () => {
           schemaName: 'Schema123456_1'
         }
       );
+
+      // The to camalCase removes the _ in the origirnal name.
+      expect(
+        refs.getSchemaNaming('#/components/schemas/schema12345_6')
```

Comment:
This means the following are treated the same, which is not optimal but since these are corner cases I would not consider for now.
- 123456
- 123_456
- 12_34_56

Reply:
yes, the `pascalCase` or `camelCase` methods will remove all this kind of formatting also small/capital letters will be removed. But I also think these are corner cases where we should not do a premature effort.